.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 .env
+lostTests
```

Comment:
@ronanfitz you should cut these off into another branch and remove this line, thanks!